### PR TITLE
initial draft for post on OSISM 7.1.0

### PR DESCRIPTION
**Pull request by fkr**

### CODE REVIEW

File: _i18n/en/_posts/blog/2024-07-12-osism-7-1-0.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  * Use of config context of the netbox for host vars
+
+From the upstream _kolla-ansible_ project, which has seen a fair share of contributions made possible by the funding through SCS, we're now able to consume the following new features:
+
+  * Parameters to disable Horizon backend & frontend
+  * Incremental MariaDB database backups
+  * org.opencontainers.image.version label now used for service versions
+  * haproxy_socket_level_admin = "yes" by default
+
+As part of our tender [for software defined storage](https://scs.community/tenders/lot3) we're working together with colleagues from [B1-Systems](https://www.b1-systems.de) on bringing
```
Comment (berendt) [inline _i18n/en/_posts/blog/2024-07-12-osism-7-1-0.md:60]: B1-Systems -> B1 Systems

**Comment by berendt** [comment]
> Final links to OSISM Release notes still need to be added

This will be the link: https://osism.tech/docs/release-notes/osism-7#710-20240710

File: _i18n/en/_posts/blog/2024-07-12-osism-7-1-0.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+category: tech
+author:
+- "Felix Kronlage-Dammers"
+avatar:
+- "fkr.jpg"
+about:
+- "fkr"
+---
+
+With the subsequent minor releases to Release 6 of the SCS IaaS reference implementation OSISM we've managed
```
Comment (garloff) [inline _i18n/en/_posts/blog/2024-07-12-osism-7-1-0.md:13]:
```suggestion
With the subsequent minor releases to Release 6 of the SCS IaaS reference implementation OSISM 7 we've managed
```
(This avoids the confusion of reading SCS R6 and OSISM 7.x later.)

**Review by garloff — APPROVED** [review]
Looks good to me, thanks.
Should we wait for the upstream release notes before merging this?

**Comment by fkr** [comment]
> Looks good to me, thanks. Should we wait for the upstream release notes before merging this?

yes! thanks for noting that they're not online yet.

**Pull request merged**